doc/conf.py

```diff
@@ -69,8 +69,8 @@
 # The short X.Y version.
 #version = '4.5'
 # The full version, including alpha/beta/rc tags.
```

Comment:
I assume this comment refers to the "release", in which case it does not match the implementation.

Reply:
I dont understand what you mean